.github/workflows/zap_export.yml

```diff
@@ -74,6 +71,9 @@ jobs:
           sudo mv ./mc /usr/bin
           mc config host add spaces $AWS_S3_ENDPOINT $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY --api S3v4
```

Comment:
Thoughts on doing a bit of zap maintenance as part of this PR?
- merge python reqs with overall repo reqs, publish new build images if needed
- use build images
- bump to ubuntu 22.04
- remove python/minio setup steps

Reply:
Ah there's a data library step below. That could be done still with a `use` step of the published action

Reply:
good idea, in-progress